repository-data/webfiles/src/main/resources/site/scss/components/_site-header.scss

```diff
@@ -1,46 +1,213 @@
 @import "../environment/tools/mixins/static-button";
+@import "../environment/tools/functions/easing";
 
-/* Site header */
-.site-header {
-    position: relative;
-    border-bottom: 1px solid $col-primary;
-    padding: round($gutter * 1.375) 0;
-    vertical-align: bottom;
+$burger-piece-width: 32px;
+$burger-piece-height: 3px;
+
+@mixin berger-menu(){
+    .list {
+        display: block;
+        margin-bottom: round($gutter * 0.5);
+
+        li {
+            display: block;
+            width: 100%;
+            margin-left: 0;
+            padding: $gutter 0;
+            border-top: 1px solid lighten($col-senary, 10%);
+
+            &:first-child {
+                border-top: none;
+            }
+        }
+    }
+}
 
-    .search-strip {
+@mixin berger-button(){
+    .main-nav__burger {
+        border: 3px solid transparent;
         position: absolute;
+        top: 5px;
         right: 0;
-        width: 50%;
+        z-index: 100;
+
+        @include media($min-width: $desktop-width, $ignore-for-ie: true) {
+            background-color: #005EB8;
+            border: 3px solid #005EB8;
+            padding: 3px 7px;
+            color: $white;
+
+            &:after {
+                content: "Menu";
+                margin-left: 5px;
+                font-size: 18px;
+                vertical-align: middle;
+            }
+        }
+
+        &:focus,
+        &:active,
+        &:hover {
+            border: 3px solid #FFB81C;
+            background-color: $white;
+            color: #005EB8;
+            outline: 0;
```

Comment:
Sorry, forgot to mention this one:

Firefox adds [their own focus outline](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Firefox) to buttons:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/1337003/79397030-0b8e5580-7f75-11ea-9641-774bf1112e7c.png">

It's not a biggie, but you do already have a good focus style. If you don't want it, you can get rid of the dotted line with something like
```css
.main-nav__burger::-moz-focus-inner {
    border: none;
}
```

Reply:
Thank you for spotting this in Firefox. I have added it.